python/lsst/meas/deblender/baseline.py

```diff
@@ -1,7 +1,3 @@
-from builtins import zip
-from builtins import str
-from builtins import range
-from builtins import object
 #!/usr/bin/env python
```

Comment:
Don't really need this shebang.

Reply:
Agreed. I also found and removed it in some other library and test files.